dlt/common/storages/live_schema_storage.py

```diff
@@ -55,6 +56,17 @@ def commit_live_schema(self, name: str) -> Schema:
             self._save_schema(live_schema)
         return live_schema
 
+    def is_live_schema_committed(self, name: str) -> bool:
+        """Checks if live schema is present in storage and have same hash"""
+        live_schema = self.live_schemas.get(name)
+        if live_schema is None:
+            raise SchemaNotFoundError(name, f"live-schema://{name}")
+        try:
+            stored_schema_json = self._load_schema_json(name)
+            return live_schema.version_hash == cast(str, stored_schema_json.get("version_hash"))
```

Comment:
is using `cast` relevant here, I mean we are not using it for any type hints and detections?